src/service/jwt.service.js

```diff
@@ -0,0 +1,46 @@
+import 'dotenv/config';
+import jwt from 'jsonwebtoken';
+import { Console } from '../utils/Console.js';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '15m',
+  });
+
+  return token;
+}
+
+function verify(token) {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (error) {
+    Console.log(error);
```

Comment:
Consider using `Console.error` instead of `Console.log` for logging errors to maintain consistency in error logging.